src/elements/content-sidebar/activity-feed/annotation-thread/__tests__/AnnotationThreadCreate.test.js

```diff
@@ -22,12 +24,22 @@ jest.mock('../../comment-form', () => props => {
         </div>
     );
 });
+jest.mock('../../../../common/annotator-context/useAnnotatorEvents', () => {
+    const mockedEmitAddAnnotationStartEvent = jest.fn();
+    const mockedEmitAddAnnotationEndEvent = jest.fn();
+
+    return jest.fn(() => ({
+        emitAddAnnotationStartEvent: mockedEmitAddAnnotationStartEvent,
+        emitAddAnnotationEndEvent: mockedEmitAddAnnotationEndEvent,
+    }));
+});
 
 describe('elements/content-sidebar/activity-feed/annotation-thread/AnnotationThreadCreate', () => {
     const mockHandleCancel = jest.fn();
     const mockOnAnnotationCreate = jest.fn();
     const mockOnError = jest.fn();
     let mockCreateAnnotation = jest.fn();
+    const mockEventEmitter = {};
```

Comment:
We typically want to group out `let` and `const` with each other if they're colocated.